test/minikube/minikube_base_restore_test.go

```diff
@@ -179,6 +180,70 @@ func TestKubernetesJournalBackupSuspended(t *testing.T) {
 	verifyBackup(t, namespaceName, admin0, "demo", &options)
 }
 
+func restoreDatabaseByArchiveType(t *testing.T, options helm.Options, namespaceName string, admin0 string, archiveType string) {
+	isLsaType := archiveType == "lsa"
+	name := "restoreDatabaseSameVersion"
+	if(isLsaType) {
+		name = "restoreLsaDatabaseSameVersion"
+	}
+
+	t.Run(name, func(t *testing.T) {
+		defer testlib.Teardown(testlib.TEARDOWN_DATABASE)
+
+		if(isLsaType) {
+			options.SetValues["database.archiveType"] = "lsa"
+		}
+
+		databaseChartName := testlib.StartDatabase(t, namespaceName, admin0, &options)
+
+		delete(options.SetValues, "database.archiveType")
```

Comment:
For this test case the "delete" wouldn't even be necessary, but I wanted to restrict the "lsa" type to a small portion of the test case just in case there are some changes in the future. I added a similar "if" statement to keep it consistent with a few lines above.